apps/example/src/components/pages/AlertsPage.tsx

```diff
@@ -291,7 +291,7 @@ const AlertsPage: React.FC = () => {
             {/* Alert Access Key Panel */}
             <Stack direction="vertical" gap={DEFAULT_MEDIUM_APP_MARGIN}>
               <Stack direction="horizontal" gap={DEFAULT_SMALL_APP_MARGIN} alignment="center">
-                <Icon color="jn-text-theme-warning" icon="warning" />
+                <Icon color="jn:text-theme-warning" icon="warning" />
```

Comment:
Please no `jn:` prefix in apps. See why in my earlier comment.
```suggestion
                <Icon color="text-theme-warning" icon="warning" />
```